packages/bpk-foundations-web/src/base/spacing.json

```diff
@@ -60,6 +60,9 @@
     "SPACING_NONE": {
       "value": "{!SPACING_NONE}"
     },
+    "SPACING_ICON_TEXT": {
+      "value": "{!SPACING_MD}"
```

Comment:
This might be tricky due to we probably want it to use the spacing functions but we can't set that as the thing here.

So you probably need to use `{!SPACING_MD_V2}` so that it aligns with all platforms 😄

Reply:
I was wondering about that, but because no one is already using it surely we can just create the new version only?
Or will our automated scripts try to change it to a function.

Also yeah I need to choose a different token for 8px then 😆